packages/zeebe-element-templates-json-schema/src/defs/groups.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "#/groups",
+  "type": "array",
+  "description": "Custom fields can be ordered together via groups.",
+  "allOf": [
+    {
+      "$ref": "examples.json#/groups"
+    }
+  ],
+  "items": {
+    "$id": "#/groups/group",
+    "type": "object",
+    "default": {},
+    "required": [
+      "id",
+      "label"
+    ],
+    "properties": {
+      "$ref": "../../../element-templates-json-schema-shared/src/defs/groups.json#/items/properties",
```

Comment:
I tried to re-utilize `element-templates-json-schema/groups.json` better here to just add the `tooltip` property but I wan't able to. I accept suggestions on how to improve this

Reply:
We had a session about this and fixed it